examples/compare_CPMG_echo.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import pyspecdata as psd
+from pyspecProcScripts import lookup_table, select_pathway
+
+def echo_interleave(d, phcycdim):
+    "interleave even and odd echoes coming from phcycdim"
+    assert d.get_ft_prop(phcycdim)
+    retval = d.shape
+    retval.pop(phcycdim)
+    retval = retval.alloc(dtype=np.complex128, format=None)
+    retval["nEcho", 0::2] = d["ph1", +1]["nEcho", 0::2]
+    retval["nEcho", 1::2] = d["ph1", -1]["nEcho", 1::2]
+    retval.copy_axes(d).copy_props(d)
+    return retval
+    
+with psd.figlist_var() as fl:
+    for thisfile, exp_type, nodename, manystep_cpmg, thislabel, thisbasename in [
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_13",
+            False,
+            "CPMG simple cyc SW = 16",
+            "no power",
+        ),
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_19",
+            True,
+            "CPMG large cyc SW = 10.0",
+            "no power",
+        ),
+        (
+            "240702_13p5mM_TEMPOL_echo.h5",
+            "ODNP_NMR_comp/Echoes",
+            "echo_12",
+            False,
+            "echo large cyc SW = 10.0",
+            "no power",
+        ),
```

Comment:
can you add all the powers that you acquired?  I'm assuming you can diff against the earlier version where they were commented out.